src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1118,7 +1119,7 @@ void ApplicationManagerImpl::OnDeviceSwitchingStart(
       hmi_apis::FunctionID::BasicCommunication_UpdateDeviceList;
   so_to_send[jhs::S_PARAMS][jhs::S_MESSAGE_TYPE] =
       hmi_apis::messageType::request;
-  so_to_send[jhs::S_PARAMS][jhs::S_PROTOCOL_VERSION] = 3;
+  so_to_send[jhs::S_PARAMS][jhs::S_PROTOCOL_VERSION] = 2;
```

Comment:
why change S_PROTOCOL_VERSION this from 3 to 2?
why it is hardcoded, not from a global macro?

Reply:
@yang1070 
Basically I agree however I see that same approach currently is being used in multiple places. So I would suggest to change this generally later.